scripts/Events.ts

```diff
@@ -0,0 +1,111 @@
+import { EventDBManager } from "../test/EventDBManager";
+import { ethers } from "hardhat";
+import type { UniswapV3Pool2 } from "../typechain";
+
+enum EventType {
+  MINT = 1,
+  BURN = 2,
+  SWAP = 3,
+}
+
+let liquidityEventDB: EventDBManager;
+let swapEventDB: EventDBManager;
+let poolAddress = "0x8ad599c3A0ff1De082011EFDDc58f1908eb6e6D8";
+let fromBlock = 12777394;
```

Comment:
Is it possible to make these two variables (fromBlock, toBlock) parameters that the user can pass in?

Reply:
Sure, now the script is for our urgent backtest use. We can improve that by defining proper structure of events persistence so that different pools will be supported, and offering user-friendly apis for users to create and sync events data.